data-reporting-service/investigation-service/src/main/resources/application.yaml

```diff
@@ -0,0 +1,36 @@
+spring:
+  kafka:
+    stream:
+      input:
+        investigation:
+          topic-name: cdc.nbs_odse.dbo.Investigation
+        defaultData:
+          topic-name: cdc.nbs_odse.dbo.Default.input
+      output:
+        investigation:
+          topic-name: cdc.nbs_odse.dbo.Investigation.output
+          topic-name-transformed: cdc.nbs_odse.dbo.Investigation.output-transformed
```

Comment:
this might be just `nbs_investigation` to be in sync with the kafka sink connector to match with the RDB table

Reply:
As we are moving this to a common utility module, we can fix the names there